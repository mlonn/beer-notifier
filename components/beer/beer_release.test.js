const { getBeerReleases } = require("./beer_release");

const getBeerReleasesPromisified = (...args) => {
  return new Promise((resolve, reject) => {
    getBeerReleases(...args, resolve);
  });
};

test("Can fetch data from api", async () => {
  const from = new Date("2020-10-28");
  const to = new Date("2020-10-28");
  const releases = await getBeerReleasesPromisified(from, to);
  expect(releases).toMatchSnapshot()
});
