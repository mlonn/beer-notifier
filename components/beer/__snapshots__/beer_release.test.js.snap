// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can fetch data from api 1`] = `
Array [
  Object {
    "beers": Array [
      Object {
        "additional_name": "Enjoy After 07.04.16",
        "alcohol": 7,
        "alcoholPercentage": 7,
        "assortment": "BS",
        "assortmentText": "Ordervaror",
        "beer_type": "Ale",
        "bottleTextShort": "Flaska",
        "category": null,
        "categoryLevel1": "Öl",
        "categoryLevel2": "Ale",
        "categoryLevel3": "Imperial/Dubbel IPA",
        "categoryLevel4": null,
        "color": null,
        "country": "USA",
        "customCategoryTitle": "Öl, Ale, Imperial/Dubbel IPA ",
        "ethicalLabel": null,
        "grapes": Array [],
        "images": Array [],
        "isCompletelyOutOfStock": false,
        "isDiscontinued": false,
        "isEthical": false,
        "isKosher": false,
        "isManufacturingCountry": true,
        "isNews": false,
        "isOrganic": false,
        "isRegionalRestricted": false,
        "isSupplierTemporaryNotAvailable": false,
        "isTemporaryOutOfStock": false,
        "isWebLaunch": false,
        "name": "Stone Brett IPA",
        "nr": "8164001",
        "originLevel1": null,
        "originLevel2": null,
        "otherSelections": null,
        "packaging": "Flaska",
        "price": 159.9,
        "producer": "Stone Brewing Escondido",
        "producerName": "Stone Brewing Escondido",
        "productId": "24531674",
        "productLaunchDate": "2020-10-28T00:00:00",
        "productNameBold": "Stone Brett IPA",
        "productNameThin": "Enjoy After 07.04.16",
        "productNumber": "8164001",
        "productNumberShort": "81640",
        "recycleFee": 0,
        "restrictedParcelQuantity": 0,
        "sales_start": "2020-10-28T00:00:00",
        "seal": Array [],
        "stock": 0,
        "style": "Imperial/Dubbel IPA",
        "sugarContent": 0,
        "supplierName": "Cask Sweden AB",
        "taste": null,
        "tasteClockBitter": 0,
        "tasteClockBody": 0,
        "tasteClockCasque": 0,
        "tasteClockFruitacid": 0,
        "tasteClockGroupBitter": null,
        "tasteClockGroupSmokiness": null,
        "tasteClockRoughness": 0,
        "tasteClockSmokiness": 0,
        "tasteClockSweetness": 0,
        "tasteClocks": Array [],
        "tasteSymbols": Array [],
        "type": "Ale",
        "usage": null,
        "vintage": "2016",
        "volume": 750,
        "volumeText": "750 ml",
      },
    ],
    "id": "2020-10-28T00:00:00",
  },
]
`;
